tools/getkeys/index.js

```diff
@@ -3,16 +3,16 @@
  * Licensed under the MIT License.
  */
 
-const fs = require("fs");
-const os = require("os");
-const path = require("path");
-const util = require("util");
-const { exec } = require('child_process');
-const msRestAzure = require("ms-rest-azure");
-const keyVault = require("azure-keyvault");
-const rcTools = require("@fluidframework/tool-utils");
+import { appendFile as _appendFile } from "fs";
+import { homedir } from "os";
+import { join, basename, extname } from "path";
+import { promisify } from "util";
+import { exec } from "child_process";
```

Comment:
Bringing up my comment from the previous PR to import these (the native node modules) by their name, like so:
```typescript
import fs from "fs";
import os from "os";
import path from "path";
import util from "util";
import child_process from "child_process";
```

For consistency with how we use them in the rest of the repo.

Reply:
Right. They were imported and not used. Removing...